translations/Update code.txt

```diff
@@ -0,0 +1 @@
+pylupdate5 ./../InitGui.py ./../fCmd.py ./../CPipe.py ./../CUtils.py ./../dodoDialogs.py ./../dodoPM.py ./../fForms.py ./../fObservers.py ./../CFrame.py ./../pFeatures.py ./../pCmd.py  -ts ./dodo_es-ES.ts
```

Comment:
`Update code.txt` ?  
@EdgarJRobles Is this file name auto-generated or what you named it?

Reply:
It's a comamnd that use to upgrade and generate TS file from python files.

Reply:
It comes from Pyqt5

Reply:
> `Update code.txt` ?
> @EdgarJRobles Is this file name auto-generated or what you named it?

It is not auto-generated, i do it manually